src/Expect.elm

```diff
@@ -35,79 +35,96 @@ module Expect
   - [`atLeast`](#atLeast) `(arg2 >= arg1)`
   - [`true`](#true) `(arg == True)`
   - [`false`](#false) `(arg == False)`
-  - [`within`](#within) `(float equality)`
-  - [`notWithin`](#notWithin) `(float inequality)`
+  - [Floating Point Comparisons](#floating-point-comparisons)
 
-[Floating point comparison guide](#floating-point-comparisons)
 
 ## Basic Expectations
 
 @docs Expectation, equal, notEqual, all
 
 
-## Comparisons
+## Numeric Comparisons
 
 @docs lessThan, atMost, greaterThan, atLeast
 
 
-### Floating point comparisons
+### Floating Point Comparisons
```

Comment:
Since this is no longer a sub-header under Comparisons, it should be at the H2 level as well :)